### DISCUSSION TITLE
Sending the rewards of penalized identities to the zero wallet

### DISCUSSION THREAD

**Discussion by Zen-44** [idena-network/idena-docs#146] [IIPs]
Currently, the rewards of penalized identities (during the ceremony, by flip reports) are redistributed to other identities which are not reported. 
Before report rewards were implemented, this acted as a form of passive reward since everyone received a higher payout after the validation. This is now insignificant compared to the report reward mechanism which is far better suited for rewarding people who correctly report flips.

Instead of redistributing those funds, they could be burnt, similar to how mining penalties work. With this change, people will no longer be rewarded with an increase of coins as rewards, but with a decrease in the coin's circulating supply.
This change fights the inflationary aspect of the current model and helps to limit the growth of the circulating supply.

Note: The IIP was renamed due to changes after community suggestions.

**Comment by NPC69** [comment]
I'll take the risk of sounding stupid for the sake of getting to the bottom of things, but what's the big difference? If the coins are burnt then that functions essentially the same as the distribution of the coins from reported Identities. Whereas I think that the current method is better because that means more value to the identities participating in the validation and the maintenance of the network while in the other case it benefits everyone, even people who are just passively holding coins.

**Reply by Zen-44** [reply]
The redistribution model has a more palpable effect (you can see your coins in your wallet), but it also helps with the depreciation of the coin. (Note: the point is NOT to make the issuance of iDNA deflationary)
Burning the coins, while maybe not so apparent at first, still redistribute the value in the ecosystem. However, burning fights the depreciation of the coin. 

So over time, the same value is redistributed, but (consider long periods of time):
- (1) Simple redistribution of coins => there are more coins with less value
- (2) Burning the coins => there are fewer coins with more value
If we consider a fixed reward of coins per epoch, the reward gradually decreases with inflation: so the decrease is faster for the first case (1) compared to the second case (2).
Considering a fixed minting cap, burning the coins benefits later adopters since the value resides in the coin and not in the number of coins.

Putting it shortly, I think redistributing the coins does an effective job of immediately redistributing the value but in the long term it contributes to inflation. The effect of burning coins redistributes the value over time making the coin more valuable.

---

On a different note, from the perspective of reward distribution, currently, there is no difference between the number of reported identities and the number of identities that missed the validation. They are not taken into consideration for the rewards distribution even though they participated in the ceremony. 
This is unlike mining penalties where a penalized miner has an impact compared to someone who does not mine, they take away part of others' rewards and burn them (while at the same time giving it back to everyone and helping the network be more decentralized). Identities that successfully go through the validation process could also have an impact on the final distribution.

**Comment by ltraveler** [comment]
Quite an interesting idea!
We have reached the point where people sell their dividends after validation instead of depositing them to stake, which leads to a significant drop in prices. Something similar happened during the time when the farm business was quite popular, and they would sell their yields. I think something similar is happening now. If the coins were burned, that could provide a good remedy against inflation.

**Comment by cryptogeek555** [comment]
I like this idea and would support this proposal if the Oracle voting will be launched.

**Comment by Toni-d-e-v** [comment]
I like how @Zen-44  is thinking and will support this proposal when is launched.

**Comment by slavko989** [comment]
Love the proposal, and will support ANY of the future decisions that stop excessive coin emissions.
The network is generating 51,480 coins a day. A day!
What do people expect to happen? The price to rise? 
Let me do the math for you, that's:
1.544.400 coins a MONTH,
18.532.800 coins a YEAR.

There is no max supply, it will just keep generating indefinitely, and there is already a supply of 100.000.000 coins (36.000.000 is a premine).

So, yes, the team should think in a direction of cutting the excessive emission of coins soon.

**Reply by NPC69** [reply]
Since the daily emission of coins is unchanging, with each passing day it becomes lesser in number relative to the whole supply. Without constant emission, there is no incentivisation for people to participate in the maintenance of the network. A giant portion of the supply is locked away in users' stakes which provides scarcity. The more people join the network, the more scarce and harder to get Idena is. Burning coins is always good but what you've just said and the "math" you've done is not an argument at all, since the model on which Idena functions is a constantly decreasing inflation.

**Reply by NPC69** [reply]
1mil seconds is 12 days
1bil seconds is 32 years
There are coins that have multiple billions of coins in circulation. If we're issuing 1 iDNA per 1 second, getting from 1bil to 2bil coins would take another 32 years, which means tedious growth. The total supply becomes huge but the daily emission remains the same. Since a giant portion of the funds are staked people are forced to split the daily emission of not very much iDNA, thus crating scarcity.

**Comment by midenaio** [comment]
@Zen-44 thank you for the proposal. 

As we wrote in [Idena's latest chronicles](https://medium.com/idena/idena-chronicles-109-111-a130e370d11c), we are ready to support the proposed change if the majority of the network votes for it.

**Comment by 0JustinMiles0** [comment]
I like the proposal although I would prefer the tokens to be sent to the treasury address rather than being burnt. I think it would yield greater benefits in the long run as we'll be able to fund more initiatives.

**Reply by teavar** [reply]
Or a part goes to the treasury address, the other gets burned

**Reply by Zen-44** [reply]
Andrew outlined the fact that sending coins to the zero wallet is equivalent to a burn (as of now) since we have no safe solution for spending money from it. However, once we have a functional funding scheme from the zero wallet, the coins accumulated there could bring benefits in terms of funding projects.

**Reply by NPC69** [reply]
Maybe we could send the "burnt" coins to the zero wallet and then potentially use those funds to pay the greedy exchanges for their stupid listings.

**Reply by 0JustinMiles0** [reply]
It's true that in the short-term, burning or funding the zero wallet will have the same effect.

The question we should answer is what option is likely to generate more value for Idena over the long-term?

Reducing the supply or funding more/bigger projects to grow the network? 

Personally, I'd certainly bet on the latter. 

As an example, I'd point to Decred that has a well-funded treasury and a very functional process to allocate these funds to proposals. Idena has the potential to have an even more functional process due the nature of its voting mechanism but lacks the funding to even motivate putting this mechanism in place.

**Reply by Zen-44** [reply]
Indeed, once the zero wallet becomes useable by the community, the additional funds that gather would be useful.

**Comment by hlolve** [comment]
This proposal change validation rewards percentage, and no stats or simulation about its changes

**Comment by ltraveler** [comment]
I think the final version of IIP-13 is misleading people by telling them that 'it is equivalent to burning them' which it is not. The money would be saved and could be unlocked by protocol changes. Alternatively, those funds could also be stolen via a Sybil attack, followed by fraudulent protocol changes.

I would agree that a real burn could help decline inflation, but why send money to a zero wallet? People who want to donate could do it by themselves. Why should we push them to do so via protocol changes?

**Reply by NPC69** [reply]
If we suffer a Sybil attack then it's altogether most probably over for all of us, regardless of where the money is or isn't.

**Reply by 0JustinMiles0** [reply]
The proposal says "At the moment, since the money in the zero wallet can not be spent, sending the penalties to it is equivalent to burning them.". Maybe it needs a bit more nuance but I wouldn't say it is misleading. It is true that the tokenomic of sending rewards to the zero wallet until the governance of the Treasury is implemented is the same as burning those rewards. Obviously, once there is a mechanism to allocate those funds it won't be the case anymore. This doesn't mean that burning rewards would create more value for token holders than increasing the funding of the Treasury (which isn't controlled by a central authority), I think it will actually be the opposite as more Treasury funding will allow to create more value for the network (which is likely to increase the demand for the token).

Your point about having the funds stolen in a sybil attack is irrelevant to the zero wallet. If a sybil attack were to happen, funds from any wallet could be stolen the same way as funds from the zero wallet could be stolen. 

A burn of the coins would only benefit current holders whereas allocating rewards to the zero wallet will benefit current and long-term holders. 

You say that the current proposal is the same as a forced donation but it's not. The tokens that are being redistributed are from lost rewards from penalized identities. With this protocol change, the rewards that are meant to not be received by penalized identities would be sent directly to the zero wallet. It's not like they will be sent to valid identities then forcibly taken from them.

**Reply by NPC69** [reply]
Or we could just leave the zero wallet as it is and use it only for special occasions when an overwhelming majority of the network wants to spend money from it on some listing or whatever because the funds can only be accessed through a hard fork. Doing a hard fork also requires 80% of the network's consent.

**Reply by hlolve** [reply]
You say that the current proposal is the same as a forced donation but it's not. The tokens that are being redistributed are from lost rewards from penalized identities. With this protocol change, the rewards that are meant to not be received by penalized identities would be sent directly to the zero wallet. It's not like they will be sent to valid identities then forcibly taken from them.

The Idena foundation payouts 10% and Zero wallet fund 2% is the same as a TAX for validation rewards funds. The 2% will not be fixed anymore, will be at least 2% of validation rewards funds

**Reply by 0JustinMiles0** [reply]
The foundation payout has nothing to do with the zero wallet fund. The governance of the zero wallet fund can only be implemented with 80% consensus of the network. It's true that it won't be fixed at 2% anymore, but I don't see how this is an issue. The source of the additional funding is from rewards from penalized identities. Regardless of whether these rewards are burned, sent to validated identities or to the zero wallet, those identities won't receive these rewards. So I don't see how you can call this proposal a TAX since it's always been the case that penalized identities shall not receive these rewards. The fact is that this proposal does not create new inflation and does not create new penalty/tax for participants of the network.

**Comment by teavar** [comment]
How about distributing the validation rewards of penalized identities to mining nodes *equally? Perhaps it would increase the count of solo noders?

*equally even if it's a pool node

**Reply by NPC69** [reply]
My greed is intensifying

**Comment by hlolve** [comment]
<!DOCTYPE html>

<html>

<body>


Total rewards | Now | IIP-13
-- | -- | --
Staking rewards | 18% | Max. 18%
Candidate rewards | 2% | 2%
Flip rewards | 15% | Max. 15%
Extra flip reward | 20% | Max. 20%
Invitation rewards | 18% | Max. 18%
Reports rewards | 15% | 15%
Idena foundation payouts | 10% | 10%
Zero wallet fund | 2% | Min. 2%
Total | 100% | 100%


</body>

</html>

If this IIP will change validation rewards structure, why was not informed in the oracle ?

**Reply by Zen-44** [reply]
The validation rewards structure is not changed. Rewards are shared the same way as before. The difference is the way penalties are handled (after the reward distribution which remains the same).

**Reply by hlolve** [reply]
Zero wallet will receive always fixed 2% of validation rewards ?

**Reply by ltraveler** [reply]
It has changed. If before we redistributed the rewards of reported identities among the rest of the participants, now that piece of cake is supposed to be sent to the zero wallet. This is precisely how rewards are divided. Never mind whether we call it penalties or donations; it is the same rewards fund.
What is really confusing me is that first, you were calling depositing the part of the reward funds into the zero wallet 'burning coins', and now you are trying to hide the changes in rewards distribution under the term 'penalties'.

**Reply by Zen-44** [reply]
The entire pool of rewards is distributed according to the percentages for each category. Penalized identities have their validation rewards taken away. The proposal affects the way this penalty is handled. Sending it to the zero wallet does not mean it gets more than 2% of the validation rewards because those coins are from penalized identities. It is a separate source of income, the initial reward distribution remains at 2%.

**Reply by ltraveler** [reply]
`Penalized identities have their validation rewards taken away.` - exactly, based on the current rules, those rewards are merged with the whole rewards cake and distributed among other members who don't have any reported flips.
Now, that IIP pushing us to donate those funds into the Zero wallet, moreover, you are not clearly explaining the real sense, hiding it under the words `burning` and `penalties`.

**Reply by hlolve** [reply]
"The entire pool of rewards" also known as "cake" is reduced except in case no ID is reported. Currently the 2% is fixed before and after validation, after IIP-13 is higher than 2% if any ID is reported

**Comment by 0JustinMiles0** [comment]
An understated benefit of increasing the funding of the Treasury is that it will create a greater incentive for the network to establish and implement the governance of the Treasury. The current fund is so underfunded (in comparison to other blockchains) that there is very little incentive for anyone to work on the governance. People complain all the time that Idena is missing some key capabilities and has very little marketing. The Treasury fund can help funding those. The core team can't do it all and contributors are not going to contribute to large initiatives pro-bono. Let's be realistic. If we want Idena to be competitive we need more funding to attract talents and great ideas. The magic of this proposal is that we would increase the Treasury fund without creating inflation.

**Reply by hlolve** [reply]
Where is "The magic of this proposal ... "

**Comment by hlolve** [comment]
Epoch 114

Validation rewards paid: 288708 iDNA
Zero wallet 2% 5774 iDNA

Considering only 1 account 0xE76DB16f1cb739a25AD3D52e8b65Bb21e925FF70
Biggest stake account redistributed 29600 to validators

With IIP-13
Zero wallet 5774 + 29600 = 35374

12,25 % of rewards

**Reply by 0JustinMiles0** [reply]
Which at current price is about $850 per epoch. 850 *24 (approximatively the number of epoch in one year) = $20,400 

Even with IIP-13, the annual budget of the treasury is only of $20,400 which is still ridiculously low in comparison to the treasury of other blockchains. Obviously that's much better than the current $3,330 annual budget without IIP-13.

**Reply by hlolve** [reply]
The problem is there is NO MAGIC is just moving funds from validators to Zero wallet, and the 2% can change in big percentage number.

**Reply by 0JustinMiles0** [reply]
This is not magic, it's about making a decision on how to best allocate the rewards from penalized identities so it benefits the long term growth of Idena (at least that's what I think this proposal should aim to accomplish). It's undeniable that Idena treasury is underfunded in comparison to other blockchains. How do you propose to solve this problem? Printing more iDNAs? Or do you think it's not even a problem at all? I'm curious to hear alternative funding mechanisms you'll suggest so Idena has a fair shot at growing its ecosystem.

**Reply by hlolve** [reply]
The original proposal was not about Zero wallet problem. Was about burn reported IDs coins. If you think the problem is Zero wallet funds, that is other problem.

**Reply by 0JustinMiles0** [reply]
Proposals are up for public debate so they can be improved. The author updated his proposal considering the initial round of feedback. If this proposal doesn't pass, modifications to the current proposal could be considered.

**Reply by hlolve** [reply]
How many proposal did you see about the possible use of Zero wallet ? We still don´t have a good governance mechanism.

**Reply by 0JustinMiles0** [reply]
As I mentioned above, the low funding of the zero wallet creates zero incentive (no pun intended ) to create a governance mechanism for it. Take the question the other way and ask yourself, if we had a governance mechanism for the zero wallet today, how could we get more funding so the governance of the wallet can actually yield some results?

I would even say that increasing the funding of the a treasury without disrupting the tokenomic is more than half of the battle to implement functional democratic funding mechanism.

**Reply by hlolve** [reply]
If coin price will increase, there will be incentives, don´t worry.

**Reply by 0JustinMiles0** [reply]
But if you run the numbers you'll see that if iDNA price 10x, the treasury would be decently funded with IIP-13 but not without IIP-13. If iDNA price 100x, the treasury will be well funded with IIP-13 but only decently funded without IIP-13. It would take iDNA price to 1,000x for the treasury to be well funded without IIP-13. Do you really want to bet the future of the network on this scenario?

**Reply by hlolve** [reply]
If you do not have rewards for people validating, the treasury is useless

**Reply by ltraveler** [reply]
Apart from losing motivation in consequence of reduction in yield, "Burning the rewards of penalized identities" and "Depositing the rewards of penalized identities" have widely different meanings. Correct me if I'm wrong, but this is misleading people (even if it was with no interested motives) by disguising one thing as another.
People who originally wanted to vote for a decline in inflation end up voting to keep a mint of iDNA.
Imagine yourself as a network member, wanting the iDNA coin to be a stronger token. The monetary contraction is a good point on that way. I am sure many people would like to vote for that through the real burning, including me.
However, if your purpose is to increase the treasury, why should you mislead the community by representing one thing as another? Everything should be transparent, with no facts/title/description spinning. It has to be described as transparent as possible.
Honestly, the phrase "At the moment, since the money in the zero wallet cannot be spent, sending the penalties to it is equivalent to burning them" sounds quite disoriented.

**Reply by 0JustinMiles0** [reply]
I agree that the title of the proposal shouldn't have used the term "burning". It's justly how the initial proposal version was named. That said, whoever read the proposal will clearly understand the nuances of the proposed mechanism. Also, keep in mind that in order to use the funds of the Treasury, 80% of the network will have to agree on how to use these funds. So as long as a minority of 20% opposes the use of these funds (which is actually probable), sending the funds to the Treasury would be akin to a burn. 

So even people that are for a straight burn should objectively vote for the proposal rather than rejected it because, this proposal would implement a mechanism that is much closer to a straight burn versus the status quo (no burn at all).

**Reply by ltraveler** [reply]
"sending the funds to the Treasury would be akin to a burn" I can't agree with that. Those funds would be considered as locked funds but not burned. The burnt coins are lost forever, but the locked ones, since the moment they are unlocked, would start to act as regular currency, which means increasing the stock of money. If you put your money into a long-term savings account, that doesn't mean that you wouldn't be able to use that money in the future. I can't agree that it's akin to a burn.

**Reply by 0JustinMiles0** [reply]
You misquoted me, please read my previous message again.

**Reply by ltraveler** [reply]
`So as long as a minority of 20% opposes the use of these funds (which is actually probable), sending the funds to the Treasury would be akin to a burn`

It would be more illustrative if you looked at this diagram, where you can see that there is a significant difference in the flow of coins between the grey zone and the green one. We could say that, at some point in history, the green zone and the grey zone may have acted in a similar way, but that doesn't mean those zones are the same. I mean, a person who is sleeping and a dead person could act similarly, but the sleepy one will eventually wake up, which you couldn't say about the deceased.

$${ \space \color{white} ✧ \color{lightblue}Total Supply \space \color{white}✧ \color{grey}Burnt \space \color{white}✧ \color{red}Staked \space \color{white}✧\color{green}Vested}$$

![image](https://github.com/idena-network/idena-docs/assets/5625337/3ebf802f-f82e-471a-af43-741cb757630a)

**Comment by QuickMythril** [comment]
if there were some active plans already for the governance or even something suggested in the IIP itself, then this would be totally different.  this is "we don't want people to sell, so we give them less" ... 🙄

let people do what they want with their coins and stop worrying about price.

i voted NO, and i hope others do the same.

**Reply by 0JustinMiles0** [reply]
Treasury governance has just been added to the roadmap.

**Comment by 0JustinMiles0** [comment]
Here is a ranking of the Treasury from various blockchains: https://defillama.com/treasuries

As a point of comparison, Gitcoin has 3,222x more in treasury than Idena ($58m vs $18k).

**Reply by teavar** [reply]
gitcoin marketcap = $73,660,605
https://coinmarketcap.com/currencies/gitcoin/
